### DISCUSSION TITLE
可以在组件里面使用这个吗

### DISCUSSION THREAD

**Discussion by itsdapi** [1977474741/spa-custom-hooks#28] [Q&A]

**Comment by 1977474741** [comment]
vue框架下支持，原生小程序里暂不支持

**Comment by itsdapi** [comment]
> vue 框架下支持，原生小程序里暂不支持

好吧 我目前就是原生：（
希望能尽快支持吧

**Comment by 1977474741** [comment]
> > vue 框架下支持，原生小程序里暂不支持
> 
> 好吧 我目前就是原生：（ 希望能尽快支持吧

好的，暂时可以用父页面通知子组件的方式

**Comment by itsdapi** [comment]
> 

父页面向组件传数据然后在组件设一个observer吗

**Comment by itsdapi** [comment]
> > > vue 框架下支持，原生小程序里暂不支持
> > 
> > 
> > 好吧 我目前就是原生：（ 希望能尽快支持吧
> 
> 好的，暂时可以用父页面通知子组件的方式

哦哦我好像知道了 在父组件用triggerevent 触发组件的事件

**Comment by 1977474741** [comment]
> > > > vue 框架下支持，原生小程序里暂不支持
> > > 
> > > 
> > > 好吧 我目前就是原生：（ 希望能尽快支持吧
> > 
> > 
> > 好的，暂时可以用父页面通知子组件的方式
> 
> 哦哦我好像知道了 在父组件用triggerevent 触发组件的事件

对

**Comment by itsdapi** [comment]
> > > > > vue 框架下支持，原生小程序里暂不支持
> > > > 
> > > > 
> > > > 好吧 我目前就是原生：（ 希望能尽快支持吧
> > > 
> > > 
> > > 好的，暂时可以用父页面通知子组件的方式
> > 
> > 
> > 哦哦我好像知道了 在父组件用 triggerevent 触发组件的事件
> 
> 对
解决了 直接获取组件实例 然后执行里面的函数就可以了xDDDD
然后父组件用的还是这个🪝

**Comment by 1977474741** [comment]
> 

对，可以
